tests/jax/test_fused_attn.py

```diff
@@ -217,7 +267,7 @@ def test_forward(self):
         """
         Test forward without JIT
         """
-        self._setup_inputs()
+        self._setup_inputs(True)
```

Comment:
Not sure why we need this

Reply:
Yes, leftover from debugging. I'll be doing a lot of code cleanup once the CI failures are fully fixed and will re-request a review at that time.